src/Security/Certificate.cs

```diff
@@ -436,63 +440,58 @@ internal static bool Equals (SecCertificate first, SecCertificate second)
 		}
 
 #if MONOMAC
```

Comment:
Is this right? There is a nested `#if MONOMAC` inside this one (always true) and an `#if MONOMAC` inside the `#else` (always false).

Reply:
I think that's a consequence of the attributes and the way we do Deprecated -> UnsupportedOSPlatform.  

We always do:

`[Deprecated (PlatformName.MacOSX, 10,13)]`
		
to
		
```
#if NET
		[UnsupportedOSPlatform ("macos10.13")]
#if MONOMAC
		[Obsolete ()]
#endif
#else
		[Deprecated (PlatformName.MacOSX, 10,13...)]
#endif
```

It looks a bit silly inside a #if MONOMAC block, but I'd just leave it, though simplifying by hand is reasonable.

Reply:
@stephen-hawley Good catch! I think I am in favor of leaving it here since the structure is a little stuffy here and if we end of removing/changing things in the future, only the Monomac will stay obsoleted.